src/ProductConstructionService/ProductConstructionService.Api/PcsStartup.cs

```diff
@@ -145,7 +150,7 @@ internal static void ConfigurePcs(
         builder.Services.AddGitHubTokenProvider();
         builder.Services.AddScoped<IRemoteFactory, DarcRemoteFactory>();
         builder.Services.AddSingleton<Microsoft.Extensions.Internal.ISystemClock, Microsoft.Extensions.Internal.SystemClock>();
-        builder.Services.AddSingleton<ExponentialRetry>();
+        builder.Services.AddSingleton<Microsoft.DotNet.Services.Utility.ExponentialRetry>();
```

Comment:
Is there a conflict with one of the new packages? I usually prefer `using ExponentialRetry = Microsoft.DotNet.Services.Utility.ExponentialRetry`, not sure what's the better way to do it tho

Reply:
There is, the package that adds the extension method for Redis authentication is creating a conflict here.

I just saw this notation used above, so followed suit.

Reply:
I removed the conflicting import and reverted this change